helm/content-fluentd/templates/daemonset.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{ .Values.service.name }}
+  namespace: {{ .Values.service.namespace }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: {{ .Values.service.name }}
+spec:
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.service.name }}
+    spec:
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
+      containers:
+      - name: {{ .Values.service.name }}
+        image: "{{ .Values.image.repository }}:{{ .Chart.Version }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        resources:
+{{ toYaml .Values.resources | indent 12 }}
+        ports:
+        - name: http-input
+          containerPort: 8080
+          protocol: TCP
+        livenessProbe:
+          httpGet:
```

Comment:
Not sure if we should put the httpGet as a Liveness probe. The effect of it failing will be that the pod is restarted.
We usually put it as a Readiness probe, and as a Liveness probe we try a TCP connection on the HTTP port.

Reply:
Sure - I stole this from the kubernetes fluentd config, but happy to change